fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,160 @@
+import time
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from typing import Any
+
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+metrics_list = ['accuracy']
+
+
+def create_tags(**tags_in):
+    tags = []
+
+    tags.append("project:" + tags_in['project'])
+    tags.append("model:" + tags_in['model'])
+    if len(tags_in) > 2:
+        tags.append("feature:" + tags_in['feature'])
+
+    return tags
+
+
+class FiddlerCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'fiddler'
+
+    def __init__(self, name, init_config, instances):
+        super(FiddlerCheck, self).__init__(name, init_config, instances)
+
+        # Connection parameters for Fiddler. Will be parameterizing this later.
+        self.URL = 'https://demo.fiddler.ai'
+        self.ORG_ID = 'demo'
+        self.AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+        self.log.info("Inside init method.")
+        self.client = fdl.FiddlerApi(url=self.URL, org_id=self.ORG_ID, auth_token=self.AUTH_TOKEN)
+
+    #        self.base_url = self.instance.get('url')
+    #        self.api_key = self.instance.get('fiddler_api_key')
+    #        self.org = self.instance.get('organization')
```

Comment:
```suggestion
        self.base_url = self.instance.get('url')
        self.api_key = self.instance.get('fiddler_api_key')
        self.org = self.instance.get('organization')
        
        self.client = fdl.FiddlerApi(url=self.base_url, org_id=self.org, auth_token=self.api_key)
```

You can use `self.instance.get('<config_option>')` to get the config values from the conf.yaml file
See doc: https://datadoghq.dev/integrations-core/base/basics/#check-initializations